core/src/test/scala/com/nvidia/spark/rapids/tool/profiling/AutoTunerSuite.scala

```diff
@@ -749,6 +751,74 @@ class AutoTunerSuite extends FunSuite with BeforeAndAfterEach with Logging {
     assert(expectedResults == autoTunerOutput)
   }
 
+  test("AutoTuner detects non UTF-8 file-encoding") {
+    // When system properties has an entry for file-encoding that is not supported by GPU for
+    // certain expressions. Then the AutoTuner should show generate a comment warning that the
```

Comment:
Nit: Then the AutoTuner should generate a comment warning that the

Reply:
Done!